### PR TITLE
fix allow  bar to resize till start place

### PR DESCRIPTION
**Pull request by Julious1994**
Hi,
I also got below issue.
https://github.com/intljusticemission/react-big-calendar/issues/1007
I tried to solve it.
you can accept pull request if it is right for this code.

Thanks
Amit Patel

### CODE REVIEW

File: src/addons/dragAndDrop/WeekWrapper.js [hunk 1]
```diff
@@ -155,7 +155,7 @@ class WeekWrapper extends React.Component {
         return
       }
 
-      start = dates.min(dates.add(end, -1, 'day'), start)
+      start = dates.min(dates.add(end, 0, 'day'), start)
```
Comment (jquense) [inline src/addons/dragAndDrop/WeekWrapper.js]: i think this is the same as `end`? does adding nothing have an effect here?

Comment (Julious1994) [reply src/addons/dragAndDrop/WeekWrapper.js]: when we decrease a bar that time it will make possible to decrease till single day bar (same as start day).

Comment (Julious1994) [reply src/addons/dragAndDrop/WeekWrapper.js]: yes it is same as end. When we want to set bar for one day that time start and end will be same i think.
So review it and accept if it is ok.

Comment (jquense) [reply src/addons/dragAndDrop/WeekWrapper.js]: if its the same, lets remove the `dates.add()` and just use `end` same for the line above with `start`

Comment (Julious1994) [reply src/addons/dragAndDrop/WeekWrapper.js]: yes we can. If we can create new branch or something like for beta/alpha version so main version would not affect. May be we can get an issue after this change.

Comment (morrisgwilliam) [reply src/addons/dragAndDrop/WeekWrapper.js:158]: has this issue been resolved? I am having the same bug. Cannot shring the event to one day unless the start date (left drag) has been modified. then the event is allowed to reduce to one day.

**Pull request closed without merging**